src/dirty_lemmas.k.md

```diff
@@ -0,0 +1,2 @@
+```k
```

Comment:
Are you leaving this in on purpose?

I would prefer to  have no dirty lemmas before we ship.

Reply:
I just left it so that we wouldn't have to keep adding it back (and forgetting it)